homeassistant/components/xiaomi_miio/number.py

```diff
@@ -0,0 +1,214 @@
+"""Support for Xiaomi Mi Air Purifier and Xiaomi Mi Air Humidifier."""
+from dataclasses import dataclass
+from enum import Enum
+import logging
+
+from homeassistant.components.number import NumberEntity
+from homeassistant.const import CONF_HOST, CONF_TOKEN
+from homeassistant.core import callback
+
+from .const import (
+    CONF_DEVICE,
+    CONF_FLOW_TYPE,
+    CONF_MODEL,
+    DOMAIN,
+    FEATURE_FLAGS_AIRHUMIDIFIER,
+    FEATURE_FLAGS_AIRHUMIDIFIER_CA4,
+    FEATURE_FLAGS_AIRHUMIDIFIER_CA_AND_CB,
+    FEATURE_SET_MOTOR_SPEED,
+    KEY_COORDINATOR,
+    KEY_DEVICE,
+    MODEL_AIRHUMIDIFIER_CA1,
+    MODEL_AIRHUMIDIFIER_CA4,
+    MODEL_AIRHUMIDIFIER_CB1,
+    MODELS_HUMIDIFIER,
+    SERVICE_SET_MOTOR_SPEED,
+)
+from .device import XiaomiCoordinatedMiioEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_MOTOR_SPEED = "motor_speed"
+ATTR_ACTUAL_MOTOR_SPEED = "actual_speed"
+
+SERVICE_TO_METHOD = {
```

Comment:
Do we already know now that this will expand? Otherwise it seems not needed.

Reply:
Removed it